docs/csharp/whats-new/tutorials/snippets/staticinterfaces/RepeatSequence.cs

```diff
@@ -3,6 +3,10 @@
     private const char Ch = 'A';
     public string Text = new string(Ch, 1);
 
+    public RepeatSequence()
```

Comment:
This was to account for https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%207#6.